contrib/fwupd.spec.in

```diff
@@ -449,6 +451,7 @@ done
 %{_libdir}/fwupd-plugins-%{fwupdplugin_version}/libfu_plugin_linux_swap.so
 %{_libdir}/fwupd-plugins-%{fwupdplugin_version}/libfu_plugin_linux_tainted.so
 %if 0%{?have_msr}
+%{_libdir}/fwupd-plugins-%{fwupdplugin_version}/libfu_plugin_intel_spi.so
```

Comment:
I deliberately kept this off in the Fedora build as it does unholy things with the PCI bus.

Reply:
so if it's moved to a tristate option - do we perhaps want a new boolean for `experimental_plugins`?

Reply:
I don't think you can clump them together as experimental. e.g. in Fedora I enable the bluez plugin, but have it disabled in RHEL.